src/core/main.c

```diff
@@ -1364,7 +1363,6 @@ int main(int argc, char *argv[]) {
 
                 /* Running outside of a container as PID 1 */
                 arg_system = true;
-                make_null_stdio();
```

Comment:
ok get it now.

I think the container case and the host case should stay as similar as possible in their behaviour, simply to make debugging easier...